### DISCUSSION TITLE
How to stop creating multiple format of the image?

### DISCUSSION THREAD

**Discussion by MrJohnWi** [directus/directus#18996] [Feature Requests]
### Summary

I want to stop creating multiple images when I upload a image for example when I upload a image they create two images and when I use the image in display property I will create another that's increase the space in storage how to stop that so it only create one image and reuse that.

### Motivation

Need this option as it will save more space in disk

### Detailed Design

.

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

We get less optimize image

### Alternatives

Strapi

### Adoption Strategy

.

**Comment by rijkvanzanten** [comment]
Closing this as the RFC form hasn't been filled out 👍🏻 Feel free to ask questions on [Discord](https://directus.chat)!
